common/samples/ConsoleApp1/src/ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,4 @@
 ﻿using System;
-using System.Collections.Generic;
```

Comment:
The problem with the documentation is that it says "Then, create a file with these contents and save it as Program.cs in the ConsoleApp1 folder" but the contents of the file aren't what the article wants them to be.

Would it be better to create a new sample with the right contents (I'm guessing no) or remove the `literalinclude` and simply put the designed source code directly in the RST (I'd have to look up the syntax for this)?